src/Payments.Mvc/Controllers/HomeController.cs

```diff
@@ -115,6 +117,17 @@ public IActionResult Error(int code)
         public IActionResult Secure() {
             return Content("Logged in");
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [IgnoreAntiforgeryToken]
+        [Route("csp-report")]
+        public IActionResult CspReport(CspReport model)
+        {
+            Log.ForContext("report", model, true).Warning("csp-report");
+
+            return new EmptyResult();

```

Comment:
Is this like returning null or does empty result return something different?  I like it actually, since it's more descriptive

Reply:
It's a 200 with a zero length content.

Reply:
Our last audit on give, we were returning json success=true, and i got an eyebrow raise from the pen tester.